pytest_splunk_addon/splunk.py

```diff
@@ -101,81 +101,36 @@ def pytest_addoption(parser):
             " of 8.0.0. "
         ),
     )
+    group.addoption(
+        "--field-bank",
+        action="store",
+        dest="field_bank",
+        default="",
+        help="Path of the field JSON file. Refer to ReadMe for the Json format.",
```

Comment:
example here would be nice. does it accept relative and absolute paths? 
also is the file named ReadMe or README? and Json can be lower cased, but more importantly is there a json schema to describe the format.

Reply:
Support for field-bank & readme will be done in future PRs. Removed the option in this one.